### ISSUE TITLE
[EN][Quest][Netherstorm]On Nethery Wings

### ISSUE THREAD

**Issue by Luna-maria** [Atlantiss/BugTracker#7734]
http://www.wowhead.com/quest=10438/on-nethery-wings

the quest seems to work fine until you actually start to bomb the Void Conduit. The bombs do hit it, but immediately it resets its health making the quest impossible to do :( 

http://www.wowhead.com/item=29778
http://www.wowhead.com/npc=20899

**Comment by TheCopacabanaMan** [comment]
Well, i was able to do this quest: just fly over the void Conduit, this one:
http://www.wowhead.com/npc=20899/void-conduit
...and continue clicking on the Phase Disruptor, this:
http://www.wowhead.com/item=29778/phase-disruptor

Quest works

**Comment by aendry** [comment]
@TheCopacabanaMan you did this using a flying mount?

**Comment by aendry** [comment]
yeah. confirm.... the quest can be compleated but not in a vanilla style..... use a flying mount go there and start bombing. need to be fixed
